Sources/ValCommand/ValCommand.swift

```diff
@@ -187,9 +187,15 @@ public struct ValCommand: ParsableCommand {
 
     // *** C++ Transpiling ***
 
+    let clangFormat = try? find("clang-format")
+    let codeFormatter: CodeTransform =
+      clangFormat != nil
+      ? clangFormatTransformation(URL(fileURLWithPath: clangFormat!))
+      : identity()
+
```

Comment:
```suggestion
    let codeFormatter: CodeTransform? = (try? find("clang-format")).map({ clangFormatTransformation(URL(fileURLWithPath: $0!)) })
```

Can be improved if `clangFormatTransformation` does the URL construction from a path.